katex.js

```diff
@@ -95,7 +95,7 @@ const renderToHTMLTree = function(
     return buildHTMLTree(tree, expression, settings);
 };
 
-module.exports = {
+export default {
```

Comment:
Is this a breaking change for CJS users?

Reply:
@rrandallcainc As for (i) `dist/katex.{min.js,.js}`, `katex` module, webpack exports an UMD module and default export is exposed, and for (ii) `katex.js`, transpilers and/or bundlers uses `interopDefault`, which imports default export when importing ES6 module, I think there'll be no breaking changes.

Reply:
@rrandallcainc I'm don't know there is a use case of directly requiring `katex.js` (from the root), e.g., `const katex = require('./node_modules/katex/katex.js);`, as it needs transpiling, but in that case, the user has to use `katex.default`, e.g., `const katex = require('./node_modules/katex/katex.js).default;`.

Reply:
Hmm, but I still think katex.js serves as the entry point, right? Even though this will be wrapped in a UMD, I think it's going to be a breaking change when requiring the built artifact. This is due to ES6 modules always being named. When CJS users try and import Katex, they'll need to change their import from `require('katex')` to `require('katex').default`. 

Perhaps I am missing something though.

Reply:
@rrandallcainc Webpack has option `libraryExport`, which configures which module or modules will be exposed via UMD and it is set to `default`. So requiring `katex` module(`require('katex')`) will expose default export and work same.